pkg/api/aim/experiments.go

```diff
@@ -36,10 +38,13 @@ func GetExperiments(c *fiber.Ctx) error {
 			"experiments.lifecycle_stage",
 			"experiments.creation_time",
 			"COUNT(runs.run_uuid) AS run_count",
+			"COALESCE(MAX(experiment_tags.value), '') AS description",
```

Comment:
`MAX` function seems a little weird here, with a string value

Reply:
Yes it is returning the string that alphabetizes to the top, but since there's only one tag with that key it is always the right description.